eng/pipelines/aggregate-reports.yml

```diff
@@ -10,6 +10,11 @@ jobs:
     pool:
       vmImage: 'ubuntu-18.04'
     steps:
+
+    - template: ../common/pipelines/templates/steps/verify-links.yml
```

Comment:
If we're doing this as part of the Analyze step for every pipeline, does this really need to be in here? I don't think this is going to provide any value here.